keras_cv/bounding_box/converters.py

```diff
@@ -198,6 +283,9 @@ def convert_format(
     - `"center_xyWH"`.  In this format the first two coordinates represent the x and y
         coordinates of the center of the bounding box, while the last two represent
         the width and height of the bounding box.
+    - `"center_yxHW"`.  In this format the first two coordinates represent the y and x
```

Comment:
The current format seems weird to me....as it contains `class` and `confidence`....I'd rather we have a good pretrained weights, and then we should probably fix this...otherwise adding `center_yxhw` format to it seems adding tech debt